man/systemctl.xml

```diff
@@ -64,7 +64,9 @@
             <replaceable>PATTERN</replaceable>s are specified, only units matching one of them are shown. The units
             that are shown are additionally filtered by <option>--type=</option> and <option>--state=</option> if those
             options are specified.</para>
-
+            <para>
+              The <option>--by-slice</option> shows the tree structure of slices for the units filtered by <option>--type</option>, <option>--state</option> or <option>--all</option>. For the tree structure to be shown, loaded units must have the root slice '-.slice'. Additionally, units can be passed as argument without any of the options stated above. For such cases, the root slice must also be passed to enable printing tree structure from the root.
```

Comment:
we have this rule that switches that take parameters should always be suffixed with a `=` in our docs. i.e. `--type=`, `--state=`